README.md

```diff
@@ -1217,6 +1217,10 @@ make "docker-build^IMPL"
   out. These dependencies are download to dot-files in the /mal
   directory so they will persist between runs.
 
+## Projects using mal
+
+ * <https://github.com/dubek/malc> - Mal (Make A Lisp) compiler. Compiles a Mal program to LLVM assembly language, then binary.
+ * <https://github.com/chr15m/frock> - Clojure-flavoured PHP. Uses mal/php to run programs.
```

Comment:
The links should use markdown link formatting (for example, see link to miniMAL at the top of the README.md file).

Reply:
Good point. Pushed a fix.